src/overlays/actors/ovl_En_Takaraya/z_en_takaraya.h

```diff
@@ -7,11 +7,21 @@ struct EnTakaraya;
 
 typedef void (*EnTakarayaActionFunc)(struct EnTakaraya*, PlayState*);
 
+#define EN_TAKARAYA_GET_SWITCH_FLAG(thisx) (u8)((thisx)->params >> 0x8)
+#define EN_TAKARAYA_GET_FF(thisx) ((thisx)->params & 0xFF)
+
 typedef struct EnTakaraya {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x164];
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ Vec3s jointTable[24];
+    /* 0x218 */ Vec3s morphTable[24];
```

Comment:
```suggestion
    /* 0x188 */ Vec3s jointTable[OBJECT_BG_1_LIMB_MAX];
    /* 0x218 */ Vec3s morphTable[OBJECT_BG_1_LIMB_MAX];
```
We're trying to avoid constants for `jointTable`/`morphTable`